### DISCUSSION TITLE
Blocklist compatibility

### DISCUSSION THREAD

**Discussion by JuanDYB** [AdguardTeam/AdGuardHome#6706] [Q&A]
Hi,

I'm new to Adguard Home, I'm trying it after using Pihole for years.
I have seen this page about [Adguard blocklist](https://adguard.com/kb/es/general/ad-filtering/adguard-filters/), but I'm not sure if they are compatible with Adguard Home because they are using more complex rules.

Thanks for your reply. With regards,

**Comment by ainar-g** [comment]
AdGuard Home uses the *AdGuard DNS filter* rule-list by default, and there is a selection of pre-approved DNS rule-lists, as well as the ability to add custom ones, of course.

**Reply by JuanDYB** [reply]
I know, but that I want to know if the blocklists in the link that I provided are compatible with AdguardHome or only with other Adguard products. I have seen the lists and have Javascipt rules and other things.
With regards,

**Reply by ainar-g** [reply]
I'm not sure I understand the question, sorry.  There are multiple filters on the page, and only some of them are DNS filters.  AdGuard Home ignores rules that aren't DNS rules, so while you can add a rule intended for the main AdGuard app, it will probably not have much effect on AdGuard Home.
